source/community/reactnative/src/components/containers/container.js

```diff
@@ -52,18 +52,8 @@ export class Container extends React.Component {
 
 	internalRenderer = () => {
 		const payload = this.payload;
-		let containerStyle;
-		let minHeight = payload.minHeight;
-		if (minHeight) {
-			if (typeof (minHeight) == 'string') {
-				const lastIndex = minHeight.lastIndexOf('px');
-				minHeight = lastIndex != -1 ? parseInt(minHeight.substring(0, lastIndex)) : minHeight
-			}
-			//If it is number, then directly we can assign minHeight
-			containerStyle = [styles.container, { minHeight }];
-		} else {
-			containerStyle = [styles.container];
-		}
+		const minHeight = Utils.convertStringToNumber(payload.minHeight);
+		const containerStyle = [styles.container, minHeight && { minHeight }];
```

Comment:
Won't this generate an array with 2 items, with the second one being `false` if `minHeight == 0` or is `null`? And if so is that ok?

Reply:
Since it is a style container. I hope, It's ok to have the condition like this.

Reply:
I have tested and verified for all the below patterns.
minHeight: "100"
minHeight: "100px"
minHeight: "100a5"
minHeight: 100  
minHeight: "ABC"
minHeight: null   
minHeight: ''ABC100"

Please let me know if I am missing any other patterns. Thanks!

Reply:
@dclaux - We need to consider only the string with "px" or also we can have a pattern like below.

eg: "100px" / "100" / "100dp"

the value for the above strings are the same - "minHeight: 100"

Reply:
Only "100px" is valid.

Reply:
BTW when you say you "hope it's ok", I'd rather you be sure :) Maybe rewriting the code like this would be better?

```typescript
const containerStyle = typeof minHeight === "number" ? [styles.container, { minHeight }] : [styles.container];
```

Reply:
Got it. Have updated the container style assignment as above.